### PR TITLE
Tweaks for acquire_iq and dep updates

### PR DESCRIPTION
**Pull request by sjkelly**
We need to drop our buffer count in acquire_iq with the MTU boost in #72. MTU went up 8x so we drop from 256 to 32 for our buffer count here.

Adjusting samplerate and bandwidth to a power of two factor _seems_ to make VCO tune fails less frequent.

Also update some deps.

image

image

### CODE REVIEW

File: software/Makefile.nvidia [hunk 1]
```diff
@@ -10,7 +10,7 @@ endif
 
 # Use our own fork until this PR is merged: https://github.com/NVIDIA/open-gpu-kernel-modules/pull/3
 NV_DRIVER_URL := https://github.com/JuliaComputing/open-gpu-kernel-modules.git
-NV_DRIVER_VERSION := sjk/rebar2
+NV_DRIVER_VERSION := sjk/rebar4
```
Comment (staticfloat) [inline software/Makefile.nvidia:13]: So it looks like this is rebased upon a newer driver version, and also reverts a commit?  Was that commit bad?  Is this change necessary for this PR?

**Review by staticfloat — APPROVED** [review]

File: software/Makefile.nvidia (hunk 1 above)
Comment (sjkelly) [reply software/Makefile.nvidia:13]: It would be good to have, since it is what has been running locally on pathfinder for months now. I only noticed the branch discrepancy last night when rebuilding. The reverted commit was some requests from Nvidia that I didn't quite get right.

Comment (staticfloat) [reply software/Makefile.nvidia:13]: Alright.  In the future, let's try to keep changes like this in a separate PR, so that it's easier to revert in the future, if we need to.

**Comment by staticfloat** [comment]
Looking good!
